utils/composer-packages/src/Command/AddComposerTypo3ExtensionsToConfigCommand.php

```diff
@@ -142,7 +142,7 @@ private function addReplacePackages(ExtensionCollection $collection): void
         $this->decorateNamesToFullyQualified($nodes);
 
         $nodeTraverser = new NodeTraverser();
-        $this->replacePackageRector->setReplacePackages($collection->getReplacePackages());
+        $this->replacePackageRector->configure($collection->getRenamePackages());
```

Comment:
This rule is configurable, so it should use `configure()` method convention + implement the `Rector\Core\Contract\Rector\ConfigurableRectorInterface` interface